File: Examples/CSharp/Programming-Documents/Fields/FormFieldsGetByName.cs
using System;
using System.Collections;
using System.IO;
using Aspose.Words;
using Aspose.Words.Fields;
namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Fields
{
    class FormFieldsGetByName
    {
        public static void Run()
        {
            //ExStart:FormFieldsGetByName
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithFields();

            Document doc = new Document(dataDir + "FormFields.doc");
            FormFieldCollection documentFormFields = doc.Range.FormFields;

            FormField formField1 = documentFormFields[3];
            FormField formField2 = documentFormFields["Text2"];
            //ExEnd:FormFieldsGetByName
            Console.WriteLine("\n" + formField2.Name + " field have following text " + formField2.GetText() + ".");
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Fields/FormFieldsGetByName.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words
Imports Aspose.Words.Fields
Public Class FormFieldsGetByName
    Public Shared Sub Run()
        ' ExStart:FormFieldsGetByName
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithFields()

        Dim doc As New Document(dataDir & Convert.ToString("FormFields.doc"))
        Dim documentFormFields As FormFieldCollection = doc.Range.FormFields

        Dim formField1 As FormField = documentFormFields(3)
        Dim formField2 As FormField = documentFormFields("Text2")
        ' ExEnd:FormFieldsGetByName
        Console.WriteLine(vbLf + formField2.Name + " field have following text " + formField2.GetText() + ".")
    End Sub
End Class
